packages/x-date-pickers/src/StaticNextDateTimePicker/StaticNextDateTimePicker.tsx

```diff
@@ -41,11 +41,11 @@ const StaticNextDateTimePicker = React.forwardRef(function StaticNextDateTimePic
     viewRenderers,
     displayStaticWrapperAs,
     showToolbar: defaultizedProps.showToolbar ?? displayStaticWrapperAs === 'mobile',
```

Comment:
@flaviendelangle @alexfauquette 
Shouldn't this prop end up in the same place we migrated the `hideTabs` prop? 🤔 

```diff
- hideTabs={false}
- showToolbar
+ slotsProps={{
+   tabs: { hidden: false },
+   toolbar: { hidden: false },
+ }}
```

Reply:
I proposed it in https://github.com/mui/mui-x/pull/6445#issuecomment-1276308006

TLDR: We agreed to do it but not on #6445 to keep the PR focused on the migration to slots.

Reply:
Created [an issue](https://github.com/mui/mui-x/issues/7472) for it. 😉